data_generation/generateWeightedRandomNum.js

```diff
@@ -0,0 +1,13 @@
+const generateWeightedRandomNum = (min, middle, max) => { // 5, 20, 50
+  var randomize = Math.floor(Math.random() * 5 + 1);
+
+  // 20% of the time, generate a random high num
+  if (randomize === 5) {
+    return Math.floor(Math.random() * (max - middle) + middle);
+  }
+  
+  // 80% of the time, generate a random low num
```

Comment:
Interesting way to tackle the issue of having too many items with a high number of reviews.